src/lib/pipeline_configuration.py

```diff
@@ -0,0 +1,1039 @@
+                       )
+                   ],
+                   ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET_SCHEME.get_code_with_match_value("covid19 adaptation kakuma s01e08"),
+                   raw_field_fold_strategy=FoldStrategies.concatenate),
+
+        CodingPlan(raw_field="rqa_s01e09_raw",
+                   dataset_name="kakuma_s01e09",
+                   time_field="sent_on",
+                   run_id_field="rqa_s01e09_run_id",
+                   coda_filename="kakuma_s01e09.json",
+                   icr_filename="kakuma_s01e09.csv",
+                   coding_configurations=[
+                       CodingConfiguration(
+                           coding_mode=CodingModes.MULTIPLE,
+                           code_scheme=CodeSchemes.KAKUMA_S01E09,
+                           coded_field="rqa_s01e09_coded",
+                           analysis_file_key="rqa_s01e09_",
+                           fold_strategy=lambda x, y: FoldStrategies.list_of_labels(CodeSchemes.KAKUMA_S01E09, x, y)
+                       )
+                   ],
+                   ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET_SCHEME.get_code_with_match_value("covid19 adaptation kakuma s01e09"),
+                   raw_field_fold_strategy=FoldStrategies.concatenate),
+
+        CodingPlan(raw_field="rqa_s01e10_raw",
+                   dataset_name="kakuma_s01e10",
+                   time_field="sent_on",
+                   run_id_field="rqa_s01e10_run_id",
+                   coda_filename="kakuma_s01e10.json",
+                   icr_filename="kakuma_s01e10.csv",
+                   coding_configurations=[
+                       CodingConfiguration(
+                           coding_mode=CodingModes.MULTIPLE,
+                           code_scheme=CodeSchemes.KAKUMA_S01E10,
+                           coded_field="rqa_s01e10_coded",
+                           analysis_file_key="rqa_s01e10_",
+                           fold_strategy=lambda x, y: FoldStrategies.list_of_labels(CodeSchemes.KAKUMA_S01E10, x, y)
+                       )
+                   ],
+                   ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET_SCHEME.get_code_with_match_value("covid19 adaptation kakuma s01e10"),
+                   raw_field_fold_strategy=FoldStrategies.concatenate),
+    ]
+
+    @staticmethod
+    def clean_age_with_range_filter(text):
+        """
+        Cleans age from the given `text`, setting to NC if the cleaned age is not in the range 10 <= age < 100.
+        """
+        age = swahili.DemographicCleaner.clean_age(text)
+        if type(age) == int and 10 <= age < 100:
+            return str(age)
+            # TODO: Once the cleaners are updated to not return Codes.NOT_CODED, this should be updated to still return
+            #       NC in the case where age is an int but is out of range
+        else:
+            return Codes.NOT_CODED
+
+    SURVEY_CODING_PLANS = None
```

Comment:
If splitting by camp do you need this?

Reply:
yes, other shared modules refer to SURVEY_CODING_PLANS instead of specific camp survey plans. We infer this in generate_outputs.py